example/webhook/webhook-example.cabal

```diff
@@ -16,7 +16,9 @@ executable webhook-example
         pusher-http-haskell -any,
         text -any,
         unordered-containers -any,
-        yaml -any
+        http-server -any,
```

Comment:
Can you use Snap for consistency with the existing server example? I also think it's probably more mainstream.